### PR TITLE
Get center coords using the /boundaries api

### LINKED ISSUE #3: Change initial coordinates
The webpage starts at Turin; it should start at the center of the used `.pbf`

### PR DESCRIPTION
**Pull request by mendaomn**
### Why

Closes #3 

### How

- gets options asynchronously by hitting the `/boundaries` api and computing the center coords
- moves map initialization into a function `initMap(options)` that gets called when the options have been fetched
- imports polyfills from https://polyfill.io/v3/ to support older browsers that lack Promises and Fetch (https://caniuse.com/#feat=fetch)

### CODE REVIEW

File: static/nearby_map.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
     -->
     <link rel="stylesheet" href="css/main.css"/>
     <link rel="stylesheet" href="resources/fontawesome-free-5.13.0-web/css/all.min.css">
+    <script crossorigin="anonymous" src="https://polyfill.io/v3/polyfill.min.js"></script>
```
Comment (mendaomn) [inline static/nearby_map.html:17]: The good thing about this is that in newer browsers the bundle is small or empty

File: static/lib/main.js [hunk 2]
```diff
@@ -13,30 +9,40 @@ var qrcode = new QRCode(qrcode_div, window.location.href);
 
 var tile_url = 'map/tile/{z}/{x}/{y}.png';
 var tile_attribution = '&copy; <a href="https://openstreetmap.org/copyright">OpenStreetMap contributors</a>';
+var map
```
Comment (mendaomn) [inline static/lib/main.js:12]: map is a globally scoped variable, as it was referenced in `lib.js`

**Review by mendaomn — COMMENTED** [review]
Added comments (:

**Comment by AlessioGiambrone** [comment]
Good work, thank you!

**Pull request merged**